voyager-publish/bin/voyager-publish.js

```diff
@@ -41,6 +43,13 @@ console.log(
 
 let rpcUrlIndex = 0
 
+const client = new pg.Pool({ connectionString: DATABASE_URL })
+
+await client.query(
+  `UPDATE measurements SET lock = NULL WHERE lock IS NOT NULL`,
+  [ps.pid]
+)
```

Comment:
We don't need the `pid` parameter. I think we can also remove the `WHERE` condition.

```suggestion
// Clear any locks left over after the previous instance exited
await client.query('UPDATE measurements SET lock = NULL')
```